### PR TITLE
Setup rspec

### PR DESCRIPTION
**Pull request by mikeymurph77**
@edwardloveall - Is this the best way to notify you for PR's?

Went through the second chapter of "Everyday Rails Testing with RSpec". In this chapter it was about setting up RSpec and configuring files.

### CODE REVIEW

**Comment by edwardloveall** [comment]
I'm now watching the repo so I'll get notifications when you post new PRs. Thanks for pinging me on this one!

File: Gemfile [hunk 1]
```diff
@@ -20,3 +22,11 @@ group :development, :test do
   gem 'better_errors'
   gem 'binding_of_caller'
 end
+
+group :test do
+  gem "faker", "~> 1.4.3"
+  gem "capybara", "~> 2.4.3"
+  gem "database_cleaner", "~> 1.3.0"
+  gem "launchy", "~> 2.4.2"
+  gem "selenium-webdriver", "~> 2.43.0"
```
Comment (edwardloveall) [inline Gemfile:31]: You may end up wanting `capybara-webkit` instead of selenium, but it'll be fine for now.

File: Gemfile [hunk 2]
```diff
@@ -20,3 +22,11 @@ group :development, :test do
   gem 'better_errors'
   gem 'binding_of_caller'
 end
+
+group :test do
+  gem "faker", "~> 1.4.3"
```
Comment (edwardloveall) [inline Gemfile:27]: I've kind of found faker to be an anti-pattern. It's not bad per se, but when I used it ended up trying to make _everything_ use fake data and it slowed me down a lot. Keep it in for now and you may find that you use it just fine.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```
Comment (edwardloveall) [inline spec/rails_helper.rb]: Let's get rid of all the comments here. They might seem helpful at first but they will become out of date and misleading. A good thing to do is link to the documentation for configuration at the top:

``` ruby
# https://relishapp.com/rspec/rspec-core/v/3-1/docs/configuration
```

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,85 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (edwardloveall) [inline spec/spec_helper.rb]: Same here with comments

File: Gemfile (hunk 2 above)
Comment (mikeymurph77) [inline Gemfile:27]: Ok, what I'll give it a try. If it's really holding me up I'll drop it.

**Pull request merged**